dist/gcc-compatible/EverCrypt_DRBG.c

```diff
@@ -225,6 +214,22 @@ EverCrypt_DRBG_state_s *EverCrypt_DRBG_create(Spec_Hash_Definitions_hash_alg a)
   return buf;
 }
 
+/**
+Create a DRBG state.
+
+@param a Hash algorithm to use. The possible instantiations are ...
+  * `Spec_Hash_Definitions_SHA2_256`,
+  * `Spec_Hash_Definitions_SHA2_384`,
+  * `Spec_Hash_Definitions_SHA2_512`, and
+  * `Spec_Hash_Definitions_SHA1`.
+
+@return DRBG state. Needs to be freed via `EverCrypt_DRBG_uninstantiate`.
+*/
+EverCrypt_DRBG_state_s *EverCrypt_DRBG_create(Spec_Hash_Definitions_hash_alg a)
+{
+  return EverCrypt_DRBG_create_in(a);
```

Comment:
Is not inlining this anymore intended? @protz

Reply:
yes. inlining it violates the CAbstractStruct premise because it results in the caller allocating a *value* of type `EverCrypt_DRBG_state_s`

Reply:
oh sorry, `create_in` is the one that cannot be inlined; this one could be conceivably inlined back

Reply:
I think I removed both instead of just one

Reply:
Probably not a big deal, but could be worth fixing in a further PR.